From 6e0ce977a45247f3ed37fc506f1834205f3c0d5f Mon Sep 17 00:00:00 2001
From: Stelios Tsampas <loathingkernel@gmail.com>
Date: Thu, 23 Apr 2026 00:55:12 +0300
Subject: [PATCH 14/14] Revert "HACK: steam: ntdll: Append C:/Program Files
 (x86)/Steam to PATH."

This reverts commit adba6db13ee035341983ee2c8e72b2440a2900c1.
---
 dlls/ntdll/loader.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dlls/ntdll/loader.c b/dlls/ntdll/loader.c
index c2f6190e59f..a55d0a525c3 100644
--- a/dlls/ntdll/loader.c
+++ b/dlls/ntdll/loader.c
@@ -87,7 +87,7 @@ const WCHAR windows_dir[] = L"C:\\windows";
 static const WCHAR system_dir[] = L"C:\\windows\\system32\\";
 
 /* system search path */
-static const WCHAR system_path[] = L"C:\\windows\\system32;C:\\windows\\system;C:\\windows;C:\\Program Files (x86)\\Steam";
+static const WCHAR system_path[] = L"C:\\windows\\system32;C:\\windows\\system;C:\\windows";
 
 #define IS_OPTION_TRUE(ch) ((ch) == 'y' || (ch) == 'Y' || (ch) == 't' || (ch) == 'T' || (ch) == '1')
 
-- 
2.54.0

